src/encoded/static/scss/encoded/modules/_regulome_search.scss

```diff
@@ -734,21 +774,28 @@ table {
     flex-wrap: wrap;
     align-items: stretch;
     justify-content: space-between;
+    @media screen and (max-width: $screen-sm-min) {
+        margin-top: 30px;
+    }
     .thumbnail {
         flex: 0 0 32%;
         max-width: 32%;
         align-items: center;
         padding: 10px 15px;
-        transition: flex 0.3s ease-in-out;
-        -moz-transition: flex 0.3s ease-in-out;
-        -webkit-transition: flex 0.3s ease-in-out;
+        transition: flex 0.3s ease-in-out, font-size 0.3s ease-in-out;
+        -moz-transition: flex 0.3s ease-in-out, font-size 0.3s ease-in-out;
+        -webkit-transition: flex 0.3s ease-in-out, font-size 0.3s ease-in-out;
         color: #333;
```

Comment:
Doesn’t _seem_ like you need the browser prefixes for transition (https://caniuse.com/#feat=css-transitions), or did you find a situation where you needed these?

Reply:
No, just a habit.